facefusion/processors/modules/age_modifier/core.py

```diff
@@ -137,32 +174,59 @@ def modify_age(target_face : Face, temp_vision_frame : VisionFrame) -> VisionFra
-	extend_face_landmark_5 = scale_face_landmark_5(face_landmark_5, 0.875)
-	extend_vision_frame, extend_affine_matrix = warp_face_by_face_landmark_5(temp_vision_frame, extend_face_landmark_5, model_templates.get('target_with_background'), model_sizes.get('target_with_background'))
-	extend_vision_frame_raw = extend_vision_frame.copy()
-	box_mask = create_box_mask(extend_vision_frame, state_manager.get_item('face_mask_blur'), (0, 0, 0, 0))
-	crop_masks =\
-	[
-		box_mask
-	]
-
-	if 'occlusion' in state_manager.get_item('face_mask_types'):
-		occlusion_mask = create_occlusion_mask(crop_vision_frame)
-		temp_matrix = merge_matrix([ extend_affine_matrix, cv2.invertAffineTransform(affine_matrix) ])
-		occlusion_mask = cv2.warpAffine(occlusion_mask, temp_matrix, model_sizes.get('target_with_background'))
-		crop_masks.append(occlusion_mask)
-
-	crop_vision_frame = prepare_vision_frame(crop_vision_frame)
-	extend_vision_frame = prepare_vision_frame(extend_vision_frame)
-	age_modifier_direction = numpy.array(numpy.interp(state_manager.get_item('age_modifier_direction'), [ -100, 100 ], [ 2.5, -2.5 ])).astype(numpy.float32)
-	extend_vision_frame = forward(crop_vision_frame, extend_vision_frame, age_modifier_direction)
-	extend_vision_frame = normalize_extend_frame(extend_vision_frame)
-	extend_vision_frame = match_frame_color(extend_vision_frame_raw, extend_vision_frame)
-	extend_affine_matrix *= (model_sizes.get('target')[0] * 4) / model_sizes.get('target_with_background')[0]
-	crop_mask = numpy.minimum.reduce(crop_masks).clip(0, 1)
-	crop_mask = cv2.resize(crop_mask, (model_sizes.get('target')[0] * 4, model_sizes.get('target')[1] * 4))
-	paste_vision_frame = paste_back(temp_vision_frame, extend_vision_frame, crop_mask, extend_affine_matrix)
-	return paste_vision_frame
+
+	if state_manager.get_item('age_modifier_model') == 'fran':
+		box_mask = create_box_mask(crop_vision_frame, state_manager.get_item('face_mask_blur'), (0, 0, 0, 0))
+		crop_masks =\
+		[
+			box_mask
+		]
+
+		if 'occlusion' in state_manager.get_item('face_mask_types'):
+			occlusion_mask = create_occlusion_mask(crop_vision_frame)
+			crop_masks.append(occlusion_mask)
+
+		crop_vision_frame = prepare_vision_frame(crop_vision_frame)
+		age_modifier_direction = numpy.array(numpy.interp(state_manager.get_item('age_modifier_direction'), [ -100, 100 ], [ -1, 1 ])).astype(numpy.float32)
+		target_age = numpy.mean(target_face.age) / 100
+		start_age_channel = numpy.full_like(crop_vision_frame[0][:1, :, :], target_age)
+		end_age_channel = numpy.full_like(crop_vision_frame[0][:1, :, :], target_age + age_modifier_direction)
+		crop_vision_frame = numpy.concatenate([ crop_vision_frame[0], start_age_channel, end_age_channel ], axis = 0)[ numpy.newaxis, ... ].astype(numpy.float32)
+		crop_vision_frame = forward(crop_vision_frame, crop_vision_frame, age_modifier_direction)
+		crop_vision_frame = normalize_vision_frame(crop_vision_frame)
+		crop_mask = numpy.minimum.reduce(crop_masks).clip(0, 1)
```

Comment:
minimum and paste block are the same, move it to the end - outside the conditions